08.ls_object/lib/long_style.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+class LongStyle
+  MARGIN = 1
+  HALF_A_YEAR = 182
+
+  def initialize(files)
+    @files = files
+  end
+
+  def draw
+    width = calc_width
+
+    print_total
+    @files.each do |file|
+      show_content(file, width)
+    end
+  end
+
+  private
+
+  def show_content(file, width)
+    print_type(file)
+    print_mode(file)
+    print file.nlink.to_s.rjust(width[:nlink]), "\s"
+    print file.uid.name.ljust(width[:uid])
+    print file.gid.name.ljust(width[:gid])
+    print file.size.to_s.rjust(width[:file_size]), "\s"
+    print_timestamp(file)
```

Comment:
```suggestion
    print type(file)
    print mode(file)
    print file.nlink.to_s.rjust(width[:nlink]), "\s"
    print file.uid.name.ljust(width[:uid])
    print file.gid.name.ljust(width[:gid])
    print file.size.to_s.rjust(width[:file_size]), "\s"
    print timestamp(file)
```

のようにprintするのはこのメソッドに集約させた方が一貫性があって良さそうな気がします。

Reply:
書き方を揃えました